statick_tool/plugins/tool/clang_format_tool_plugin.py

```diff
@@ -73,15 +74,28 @@ def scan(self, package, level):  # pylint: disable=too-many-locals, too-many-bra
                 if (line.startswith("+ ") or line.startswith("- ") or
                         line.startswith("! ")) and len(line) > 2:
                     if line[2:].strip() and line[2:].strip()[0] != "#":
-                        # pylint: disable=line-too-long
                         exc = subprocess.CalledProcessError(-1,
                                                             clang_format_bin,
-                                                            "_clang-format style is not correct. There is one located in {}. Put this file in your home directory.".
-                                                            format(format_file_name))
-                        # pylint: enable=line-too-long
+                                                            exc_msg)
                         if self.plugin_context.args.clang_format_raise_exception:
                             raise exc
 
+        except (IOError, OSError) as ex:
+            print("{}".format(exc_msg))
+            print("Error: {}".format(str(ex.strerror)))
+            if self.plugin_context.args.clang_format_raise_exception:
+                return None
+            return []
+
+        except subprocess.CalledProcessError as ex:
+            print("{} Returncode = {}".format(exc_msg, str(ex.returncode)))
+            print("Error: {}".format(ex.output))
```

Comment:
I think the `ex.output` string is the same as `exc_msg` in this case, which leads to a duplicate of that message being printed. That should be rectified. Maybe `exc_msg` is not needed in the first line?